src/archivist/recorder/repositories/mongo/dataMapper.js

```diff
@@ -16,7 +16,7 @@ export function toPersistence(record) {
 }
 
 export function toDomain(mongoDocument) {
-  const { _id, serviceId, termsType, documentId, fetchDate, mimeType, isRefilter, isFirstRecord, snapshotIds } = mongoDocument;
+  const { _id, serviceId, termsType, documentId, fetchDate, mimeType, isExtractOnly, isRefilter, isFirstRecord, snapshotIds } = mongoDocument;
```

Comment:
Shouldn't ESLint catch that one? 🤔 

```suggestion
  const { _id, serviceId, termsType, documentId, fetchDate, mimeType, isExtractOnly, isFirstRecord, snapshotIds } = mongoDocument;
```

Reply:
It's not an error, it is used for backwards compatibility. See line 29. (that's why Eslint did not cry)